src/main/kotlin/com/atlassian/performance/tools/report/api/distribution/DistributionComparator.kt

```diff
@@ -0,0 +1,113 @@
+package com.atlassian.performance.tools.report.api.distribution
```

Comment:
Ok, so `ShiftedDistributionRegressionTest` was API, so now its replacement also needs to be exposed. Makes sense 👍